src/main/java/uk/gov/mca/beacons/service/BeaconsRegistrationService.java

```diff
@@ -2,10 +2,8 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
 
 @SpringBootApplication
-@EnableJpaAuditing
 public class BeaconsRegistrationService {
```

Comment:
Removed @EnableJpaAuditing from here and moved to separate config file [here](https://github.com/mcagov/beacons-service/pull/78/files#diff-621f2620247ed1e240128f1cac72422d33addf35ba686efffed1f150bcbf3542) so that we can separate test and prod config where necessary.